src/pages/api/auth/login.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { PrismaClient } from "@prisma/client";
+import bcrypt from "bcrypt";
+import jwt, { Secret } from "jsonwebtoken";
+
+const prisma = new PrismaClient();
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "POST") {
+    return res.status(405).json({ message: "Method not allowed" });
+  }
+
+  const { email, password } = req.body;
+
+  // ? Do we need to check for empty fields
+  // ? Client-side validation should be enough
+  // if (!email || !password) {
```

Comment:
würde ich drin lassen, schadet ja nicht

Reply:
Erledigt in [deab844](https://github.com/top-22/forum/pull/14/commits/deab8447c0f39ff65214be94f11b32a650669b48)